### DISCUSSION TITLE
Are multi-edges allowed?

### DISCUSSION THREAD

**Discussion by nichtich** [pg-format/pg-formatter#42] [Q&A] [answered]
Some graph-format allow multi edges (same edge multiple times), some don't and definition of sameness may differ. As pg-formatter and blitzboard sandbox both seem to allow identical edges to occurr multiple times in one graph, I assume that multi-edges are allowed in PG data model.

**Comment by hchiba1** [comment, marked as answer]
Yes, I also assumed that multi-edges are allowed in PG data model.
